### PR TITLE
[Finishes #148029303] Adds validation for food name and calories and finishes #148029305

### PR DESCRIPTION
**Pull request by ebassity**
@case-eee @NessEFC

### CODE REVIEW

**Comment by ebassity** [comment]
now also closes #148029307 for clearing form and validations upon submit.

File: lib/scripts/index.js [hunk 1]
```diff
@@ -17,8 +17,9 @@ $(() => {
       url = $form.attr( 'action' )
       var posting = $.post( url, { name: $('#newFoodName').val(), calories: $('#newFoodCalories').val() })
       posting.done(function(data) {
-        alert('success')
+        alert('Success! Your food was created.')
         console.log(data)
+        $('#newFood').trigger("reset")
```
Comment (case-eee) [inline lib/scripts/index.js:22]: @somedayrainbows @NessEFC is it necessary to chain two done callbacks here? could you do it all in one?

**Comment by ebassity** [comment]
I refactored this to include it in the below done callback. Thanks
@case-eee!

**Comment by case-eee** [comment]
awesome @somedayrainbows 👍

**Pull request merged**